configs/coco_cond_stage.yaml

```diff
@@ -41,7 +41,7 @@ data:
         onehot_segmentation: true
         use_stuffthing: true
     validation:
-      target: taming.data.coco.CocoImagesAndCaptionsTrain
+      target: taming.data.coco.CocoImagesAndCaptionsValidation
```

Comment:
@rromb please double-check that these two changes I propose are correct

Reply:
looks good, thanks for catching this :+1: